### PR TITLE
[ADF-4680] Add documentation for dateValues in app.config.json

### PR DESCRIPTION
**Pull request by davidcanonieto**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [x] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-4680

**What is the new behaviour?**
Add missing documentation for dateValues in app.config.json

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/core/pipes/localized-date.pipe.md [hunk 1]
```diff
@@ -48,3 +49,24 @@ registerLocaleData(localeJa);
...
+ "dateValues": {
+   "defaultDateFormat": "mediumDate",
+   "defaultDateTimeFormat": "MMM d, y, h:mm",
+   "defaultLocale": "en-US"
+ }
+```
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| defaultDateFormat | string | A format to apply to date values |
```
Comment (gmandakini) [inline docs/core/pipes/localized-date.pipe.md]: A format to apply to date values => The format to apply to date values

File: docs/core/pipes/localized-date.pipe.md [hunk 2]
```diff
@@ -48,3 +49,24 @@ registerLocaleData(localeJa);
...
+| defaultDateTimeFormat | string | The format to apply to date-time values |
+| defaultLocale | string | The locale id to apply |
+|
+
+This configuration overwrites the values in the localized date pipe as well as other components to have more consistency across your app. However, you can still overwrite these values any time while using the pipe in your code.
```
Comment (gmandakini) [inline docs/core/pipes/localized-date.pipe.md]: while => by

**Comment by davidcanonieto** [comment]
24h format fix in dateTimeItem model

File: docs/user-guide/localization.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+-   [Using the time ago pipe](#using-the-time-ago-pipe)
+
+## Setting up the configuration in your app
+
+You can overwrite the default values of this pipe by adding these properties to your ```app.config.json```:
+
+```json
+ "dateValues": {
+   "defaultDateFormat": "mediumDate",
+   "defaultDateTimeFormat": "MMM d, y, h:mm",
```
Comment (DenysVuika) [inline docs/user-guide/localization.md]: maybe needs `H` here too?
```suggestion
   "defaultDateTimeFormat": "MMM d, y, H:mm",
```

File: docs/core/pipes/localized-date.pipe.md [hunk 4]
```diff
@@ -48,3 +48,23 @@ registerLocaleData(localeJa);
...
 ```
+
+### Default values
+
+You can overwrite the default values of this pipe by adding these properties to your ```app.config.json```:
+
+```json
+ "dateValues": {
+   "defaultDateFormat": "mediumDate",
+   "defaultDateTimeFormat": "MMM d, y, h:mm",
```
Comment (suzanadirla) [inline docs/core/pipes/localized-date.pipe.md]: better make the change to 'H:mm' here also

File: docs/user-guide/localization.md (hunk 3 above)
Comment (eromano) [reply docs/user-guide/localization.md]: I guess the bug that we are fixing is ok

**Pull request merged**